### PR TITLE
feat(): tab icons hover change

### PR DESCRIPTION
**Pull request by omerson**
This PR will close https://github.com/bleenco/bterm/issues/41

### CODE REVIEW

File: src/styles/window-top.sass [hunk 1]
```diff
@@ -34,6 +44,11 @@
 
       &.maximize
         background: #62C655
+        padding-top: 2px
+
+  .window-top-container__left:hover strong
```
Comment (jkuri) [inline src/styles/window-top.sass]: can you change this for separate hover, not that all icons change when hovering left corner?

File: src/app/components/window-top/window-top.component.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <div class="window-top-container" (dblclick)="tabMaximize()">
   <div class="window-top-container__left" [ngClass]="{ 'bordered': tabs.length > 1 && !tabs[0].active }">
-    <span class="icon close" (click)="close()"></span>
-    <span class="icon minimize" (click)="minimize()"></span>
-    <span class="icon maximize" (click)="maximize()"></span>
+    <span class="icon close" (click)="close()"><strong>x</strong></span>
```
Comment (jkuri) [inline src/app/components/window-top/window-top.component.html]: `X` doesn't seem to be centered

File: src/styles/window-top.sass (hunk 1 above)
Comment (omerson) [inline src/styles/window-top.sass]: OK. Makes sense.

File: src/app/components/window-top/window-top.component.html (hunk 2 above)
Comment (omerson) [inline src/app/components/window-top/window-top.component.html]: I will check.

File: src/styles/window-top.sass [hunk 3]
```diff
@@ -24,6 +24,16 @@
       border-radius: 50%
       -webkit-app-region: no-drag
       margin: 0 3px
+      line-height: 9px
```
Comment (jkuri) [inline src/styles/window-top.sass]: remove this lines and all beneath. add:
```sass
display: flex
justify-content: center
align-items: center
```

Comment (omerson) [inline src/styles/window-top.sass]: OK

**Comment by jkuri** [comment]
what's the progress on this since last review?

**Comment by omerson** [comment]
Comments are applied.

**Comment by jkuri** [comment]
for me its still working that all icons have marks inside on hover?

**Pull request merged**